salt/cloud/clouds/ec2.py

```diff
@@ -220,13 +214,8 @@ def _xml_to_dict(xmltree):
         if "}" in name:
             comps = name.split("}")
             name = comps[1]
-        if name not in xmldict:
```

Comment:
```suggestion
        if name not in xmldict:
```

Reply:
This line can't be deleted

Reply:
Wow, I'm not sure how I managed that. I've just restored the line.